samples/Azure.Management.Storage/Generated/RestOperations/BlobContainersRestOperations.cs

```diff
@@ -21,9 +21,11 @@ internal partial class BlobContainersRestOperations
     {
         private readonly string _userAgent;
         private readonly HttpPipeline _pipeline;
-        private readonly Uri _endpoint;
         private readonly string _apiVersion;
 
+        /// <summary> server parameter. </summary>
+        public Uri Endpoint { get; }
```

Comment:
Id prefer if we did this only in the public RestOperations objects.  If we update the RestClient model to add this property and remove the field only when its constructed with a public accessibility that would keep the code common but at the same time not introduce these types of unecessary changes all over the place

Reply:
If I understand the intent here.  Why wouldn't we simply add an Endpoint property to public Client and leave all of the internal classes alone?  Each of those internal classes knows about the endpoint only because its more efficient for them to cache the value, but the "owner" of that property is either the ClientOptions object for the service or the public Client for the service.  It seems adding a single property there would make more sense versus adding hundreds of properties all over the place in the internal clients that are simply copies of the original property.

Reply:
@m-nash, I tend to agree.